### DISCUSSION TITLE
Error on Start - PowerFxInterpreter

### DISCUSSION THREAD

**Discussion by MartinBrand01** [Innofactor/XrmToolBox.BulkDataUpdater#183] [General]
Hi Jonas @rappen ,
Thanks first of all for the great tool. Really helps a lot. 
Since some time, I'm getting an error after starting the tool. 

First a Message is coming up, that there is a problem loading the Microsoft.PowerFx.Interpreter 

![Error PowerFxInterpreter](https://user-images.githubusercontent.com/65211809/222497147-5fccb809-9984-4f9e-8f51-5fce070b9c3d.png)

After that, you can't start the XRM Toolbox without deleting the Assembly from C:\Users\Martin\AppData\Roaming\MscrmTools\XrmToolBox\Plugins

After deleting and trying to connect to your Environment, loading the Metadata is running forver und end in an timeout. 

Have you received any Feedback from other with an error like this? 

Thanks

**Comment by rappen** [comment]
Hi Martin,
Sorry to hear that.
Do you think you get that error as soon as you updated to version 1.2022.11.2 of BDU?

From version 1.2022.11.2 it should be included in a subfolder, can you check that?

![image](https://user-images.githubusercontent.com/2572253/222893479-a4529627-160b-4bb1-9024-eba2b32db9e6.png)

I haven't heard this issue before.
